src/components/login/LoginForm.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Card, Form, FormGroup, FormControl, Button } from 'react-bootstrap';
+const loginFormWrapperStyling = {
+    display: 'flex',
+    alignItems: 'center',
+    marginTop: -100,
+    outline:'2px solid red',
+  };
+  
+  const panelStyle = {
+    border: 0,
+    paddingLeft: 20,
+    paddingRight: 20,
+    width: 300,
+    outline:'2px solid red',
+  };
+  
+  const buttonStyle = {
+    marginBottom: 0
+  };
+
+export function LoginForm()
+{
+    return (
+      <div style={loginFormWrapperStyling}>
+        <Card style={panelStyle}>
+          <Form horizontal className="LoginForm" id="loginForm">
+            <FormGroup controlId="formEmail">
```

Comment:
Please change the target of this form to be applied for the username, as on the backend, the LoginDTO is consisted of UserName and password.